README.md

```diff
@@ -1,80 +1,161 @@
+See benchmark results [here](https://mbatoul.github.io/sklearn_benchmarks/).
 
-To consult benchmark results, see notebooks [scikit_learn_intelex_vs_scikit_learn.ipynb](https://github.com/mbatoul/sklearn_benchmarks/blob/master/scikit_learn_intelex_vs_scikit_learn.ipynb), [onnx_vs_scikit_learn.ipynb](https://github.com/mbatoul/sklearn_benchmarks/blob/master/onnx_vs_scikit_learn.ipynb) and [gradient_boosting.ipynb](https://github.com/mbatoul/sklearn_benchmarks/blob/master/gradient_boosting.ipynb).
+# Install
 
-For information regarding the benchmark environment and execution, see notebook [index.ipynb](https://github.com/mbatoul/sklearn_benchmarks/blob/master/index.ipynb).
+In order to setup the environment, you need to have `conda` installed. See instructions [here](https://conda.io/projects/conda/en/latest/user-guide/install/index.html).
 
-These notebooks automatically deployed to `github-pages`. See current results [here](https://mbatoul.github.io/sklearn_benchmarks/).
+To get a local copy up and running follow these simple example steps:
 
-sklearn_benchmarks is used through a command line as described below.
+```sh
+$ git clone -b master --single-branch https://github.com/mbatoul/sklearn_benchmarks
+$ cd sklearn_benchmarks
+$ conda env create --file dev_environment.yml
+$ conda activate sklbench_dev
+$ pip install .
+```
 
-So far, the concurrent libraries available are:
+# Quickstart
+
+The benchmark script consumes the configuration file (`config.yml` by default) which contains the specification of the scikit-learn estimators, their hyperparameters and datasets.
+
+```yml
+- name: sklearn_KNeighborsClassifier_brute_force
+  estimator: sklearn.neighbors.KNeighborsClassifier
+  strategy: hp_match
+  predict_with_onnx: True
+  parameters:
+    n_neighbors:
+      - 1
+      - 5
+      - 100
+    algorithm:
+      - brute
+    n_jobs:
+      - -1
+      - 1
+  metrics:
+    - accuracy_score
+  datasets:
+    - generator: sklearn.datasets.make_classification
+      generator_parameters:
+        n_classes: 2
+        n_redundant: 0
+      n_features: 100
+      n_samples_train:
+        - 100_000
+      n_samples_test:
+        - 1
+        - 1000
+    - generator: sklearn.datasets.make_classification
+      generator_parameters:
+        n_classes: 2
+        n_redundant: 0
+      n_features: 2
+      n_samples_train:
+        - 100_000
+      n_samples_test:
+        - 1
+        - 1000
+```
 
-- [Intel® oneAPI Data Analytics Library](https://github.com/oneapi-src/oneDAL)
-- [lightgbm](https://lightgbm.readthedocs.io/en/latest/index.html) (gradient boosting library)
-- [XGBoost](https://xgboost.readthedocs.io/en/latest/) (gradient boosting library)
-- [Catboost](https://catboost.ai/) (gradient boosting library)
-- [ONNX Runtime](https://github.com/microsoft/onnxruntime) (inferencing and training accelerator)
+**Run benchmarks for all estimators with profiling**
+```sh
+sklbench run --profile
```

Comment:
I am not sure it's worth adding a script stub in the PATH when installing the tool. I think using the `-m` flag of `python` is enough:

```
python -m sklearn_benchmarks run --profile
```